### PR TITLE
BDRE-163 fixing coding standards in jaas login

### PR DESCRIPTION
**Pull request by bdremadhav**

### CODE REVIEW

File: jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASUserPrincipal.java [hunk 1]
```diff
@@ -59,15 +59,12 @@ public int hashCode() {
     public boolean equals(Object obj) {
```
Comment (bdre-bda) [inline jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASUserPrincipal.java]: MAJOR The Cyclomatic Complexity of this method "equals" is 11 which is greater than 10 authorized. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

File: jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
 
-    private static Logger LOGGER = Logger.getLogger(JAASLoginModule.class);
+    private static final Logger LOGGER = Logger.getLogger(JAASLoginModule.class);
 
     // initial state
     private Subject subject;
     private CallbackHandler callbackHandler;
-    private Map sharedState;
-    private Map options;
+    private Map<String, ?> sharedState;
```
Comment (bdre-bda) [inline jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java:42]: MAJOR Remove this unused "sharedState" private field. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)

File: jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java [hunk 3]
```diff
@@ -34,13 +34,13 @@
...
+    private Map<String, ?> options;
```
Comment (bdre-bda) [inline jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java:43]: MAJOR Remove this unused "options" private field. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1068)

File: jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASPasswordPrincipal.java [hunk 4]
```diff
@@ -51,15 +51,13 @@ public int hashCode() {
     public boolean equals(Object obj) {
```
Comment (bdre-bda) [inline jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASPasswordPrincipal.java]: MAJOR The Cyclomatic Complexity of this method "equals" is 11 which is greater than 10 authorized. [rule](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)

**Comment by bdre-bda** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 4 major

Watch the comments in this conversation to review them.

**Comment by bdre-bda** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 4 major

Watch the comments in this conversation to review them.

**Comment by bdre-bda** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java [hunk 5]
```diff
@@ -148,9 +150,9 @@ public boolean commit() throws LoginException {
 
     @Override
     public boolean abort() throws LoginException {
-        if (succeeded == false) {
+        if (succeeded) {
             return false;
-        } else if (succeeded == true && commitSucceeded == false) {
+        } else if (succeeded && commitSucceeded) {
```
Comment (mishisinku) [inline jaas-login/src/main/java/com/wipro/ats/bdre/jaas/login/JAASLoginModule.java:155]: @bdremadhav  commitSucceeded is false add a negation !

**Comment by bdre-bda** [comment]
SonarQube analysis reported 2 issues:
- MAJOR 2 major

Watch the comments in this conversation to review them.

**Pull request merged**